views/cases/staff.hbs

```diff
@@ -1,3 +1,77 @@
-<div>
-    Staff
-</div>
\ No newline at end of file
+<div class='row justify-content-center' style="margin-top: 10px;">
+    <h3> Staff </h3>
+</div>
+
```

Comment:
Let's wrap the form markup underneath this with `<div class="card body p-2" style="border-radius: 1rem; width: 50rem; margin: auto;"></div>` and stop before u get to the table with the test data, and merge conflicts need to be resolved before this PR can be approved.  If you need any help resolving the conflicts, let me know